caluma-demo/nginx.conf

```diff
@@ -0,0 +1,55 @@
+server {
+  listen 80 default_server;
+  listen [::]:80 default_server;
+
+  root /usr/share/nginx/html;
+
+  server_name _;
+
+  proxy_set_header Host $http_host;
+
+  gzip_static on;
+  gzip_proxied any;
+
+  location / {
+    add_header Cache-Control 'private, max-age=86400';
+    try_files $uri $uri/ /index.html?/$request_uri;
+  }
+
+  location ~ ^/graphql {
+    proxy_pass http://caluma:8000;
+  }
+
+  location ~ ^/static/ {
+      proxy_pass http://caluma:8000;
+  }
+
+  location = /index.html {
+      expires epoch;
+      add_header Cache-Control 'private, no-store, must-revalidate, max-age=0';
+  }
+
+  location = /sw.js {
+      expires epoch;
+      add_header Cache-Control 'private, no-store, must-revalidate, max-age=0';
+  }
```

Comment:
I think we are not using a service worker - this is unnecessary...

Reply:
I have deleted these lines.